.docs/index.md

```diff
@@ -96,15 +93,21 @@ registered as either a SystemD or SystemV service depending upon the OS.
 $ curl -sSL https://dl.bintray.com/emccode/rexray/install | sh
 ```
 
+Refer to the [`User Guide#Installation`](./user-guide/installation/#installation) for manual builds and more.
+
+
 ### Configuring REX-Ray
-Create a configuration file on the host at `/etc/rexray/config.yml`. Here is a
-simple example for using Oracle VirtualBox EC2:
+REX-Ray requires a configuration file for storing credentials to access the storage provider. Refer to the [libStorage#Storage Providers](http://libstorage.readthedocs.io/en/stable/user-guide/storage-providers/) for sample configurations of all supported storage platforms. Additionally, look at [core properties](./user-guide/config.md#configuration-properties) & [logging](./user-guide/config.md#logging-configuration) for advanced configurations. 
```

Comment:
I'd rather state that `REX-Ray requires a settings file for storing configuration information used to connect to storage providers...` or something like that. 

I have a note about the use of `libStorage#Storage Providers` as well, but I am going to make that comment in the main conversation as it's a broader issue in scope.